applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -123,25 +123,23 @@ static void pa_synced_cb(struct bt_audio_broadcast_sink *sink, struct bt_le_per_
 static void pa_sync_lost_cb(struct bt_audio_broadcast_sink *sink)
 {
 	int ret;
-	struct event_t event;
 
 	if (broadcast_sink == NULL) {
 		LOG_ERR("Unexpected PA sync lost");
 		return;
 	}
 
-	LOG_WRN("Sink disconnected");
+	LOG_INF("Sink disconnected");
 
+	synced_to_broadcast = false;
 	broadcast_sink = NULL;
 
-	// TODO: Create static function for sending events
-	event.event_source = EVT_SRC_LE_AUDIO;
-	event.le_audio_activity.le_audio_evt_type = LE_AUDIO_EVT_NOT_STREAMING;
```

Comment:
What happens with this event? It should be sent either in the reset or in this cb

Reply:
Host calls stream_stopped_cb() before pa_sync_lost_cb()